modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -17,22 +17,31 @@ restricted network.
 
 Complete the following steps on the bastion host:
 
-. Download your `registry.redhat.io` pull secret from the
-link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
-page.
+. Generate the pull secret for your registry:
++
+----
+$ podman login --authfile ~/pullsecret_config.json <local_registry_host_name>:<local_registry_host_port> <1>
+----
+<1> For `<local_registry_host_name>`, specify the registry domain name
+for your mirror registry, such as `registry.example.com`. For
+`<local_registry_host_port>`, specify the port that your mirror registry uses to
+serve content.
++
+Provide your credentials for the mirror registry at the prompts.
 
-. Generate the base64-encoded user name and password or token for your mirror
-registry:
+. View the pull secret that you created and record the pull secret value:
 +
 ----
-$ echo -n '<user_name>:<password>' | base64 -w0 <1>
+# cat ~/pullsecret_config.json
 
-BGVtbYk3ZHAtqXs=
+{ "auths": { "<local_registry_host_name>:<local_registry_host_port>": { "auth": "ZHVtbXk6ZHVtbXk=" } } }
 ----
-<1> For `<user_name>` and `<password>`, specify the user name and password that
-you configured for your registry.
 
-. Make a copy of your pull secret in JSON format:
+. Download your `registry.redhat.io` pull secret from the
```

Comment:
Why we name the pull secret `registry.redhat.io`? The pull secret including several auths for different registries, registry.redhat.io is only one of them. Personally I think this is easy to confuse user.

Reply:
To differentiate it from the other pull secret. Will you please take a look at the new commit and let me know if it's ok now?

Reply:
LGTM.